src/components/docPage/index.tsx

```diff
@@ -54,6 +55,9 @@ export function DocPage({
 
   const leafNode = nodeForPath(rootNode, unversionedPath);
 
+  // a hack to show syntax highlighting on editors
+  const javascript = String.raw;
```

Comment:
That's a bit confusing, could we just inline this with a comment where it is used?

Reply:
we can't as we are already insde `return`, but a client component will obviate the need for this